src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1506,6 +1506,11 @@ internal static IEnumerable<string> GetAttributeStrings(ImmutableArray<CSharpAtt
             return attributes.Select(a => a.ToString());
         }
 
+        internal static IEnumerable<string> GetAttributeStrings(IEnumerable<CSharpAttributeData> attributes)
```

Comment:
>IEnumerable [](start = 64, length = 11)

nit: Can we remove the ImmuableArray overload since this overload will handle that case too?

Reply:
> Can we remove the ImmuableArray overload since this overload will handle that case too?

I do not think this would be the right thing to do. That overload allocates less.

---
In reply to: [453887146](https://github.com/dotnet/roslyn/pull/45908#discussion_r453887146) [](ancestors = 453887146)